src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -44,6 +68,69 @@ public HelpWindow() {
         this(new Stage());
     }
 
+    /**
+     * Populates the table with all the command information.
+     */
+    private void setUpTable(TableView<CommandHelper> table, ObservableList<CommandHelper> commands) {
+        TableColumn<CommandHelper, String> commandTitle = new TableColumn<>("Command Title");
+        commandTitle.setCellValueFactory(new PropertyValueFactory<>("commandTitle"));
+
+        TableColumn<CommandHelper, String> commandUsage = new TableColumn<>("Command Usage");
+        commandUsage.setCellValueFactory(new PropertyValueFactory<>("commandUsage"));
+
+        table.setItems(commands);
+
+        table.setSelectionModel(null);
+        table.prefHeightProperty()
+                .bind(Bindings.size(table.getItems()).multiply(TABLE_ITEM_HEIGHT).add(ADDITIONAL_MARGIN));
+
+        commandTitle.prefWidthProperty().bind(table.widthProperty().multiply(0.20));
+        commandUsage.prefWidthProperty().bind(table.widthProperty().multiply(0.70));
+
+        table.getColumns().add(commandTitle);
+        table.getColumns().add(commandUsage);
+    }
+
+    private static ObservableList<CommandHelper> getGeneralCommands() {
+        return FXCollections.observableArrayList(
+                new CommandHelper("Open help panel", "help"),
+                new CommandHelper("List all", "list"),
+                new CommandHelper("Clear all data", "clear"),
+                new CommandHelper("Exit", "exit")
+        );
+    }
+
+    private static ObservableList<CommandHelper> getStudentCommands() {
+        return FXCollections.observableArrayList(
+                new CommandHelper("Add student", "add_student n/NAME p/STUDENT_PHONE_NUMBER "
+                        + "e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN"),
+                new CommandHelper("Find student", "find_student KEYWORD [MORE_KEYWORDS]"),
+                new CommandHelper("Edit student", "edit_student STUDENT_INDEX [n/NAME]"
+                        + "[p/STUDENT_PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [l/STUDY_LEVEL] [g/GUARDIAN_PHONE_NUMBER]"
+                        + " [r/RELATIONSHIP_WITH_GUARDIAN]"),
+                new CommandHelper("Delete Student", "delete_student STUDENT_INDEX"),
+                new CommandHelper("List emails", "emails")
+        );
+    }
+
+    private static ObservableList<CommandHelper> getSessionCommands() {
+        return FXCollections.observableArrayList(
+                new CommandHelper("Add single session", "add_session n/STUDENT_NAME d/DATE "
+                        + "t/TIME k/DURATION s/SUBJECT f/FEE"),
+                new CommandHelper("Add recurring session", "add_rec_session n/STUDENT_NAME "
+                        + "d/DATE e/LASTDATE b/INTERVAL t/TIME k/DURATION s/SUBJECT f/FEE"),
```

Comment:
Think this concerns the UG as well.
In the individual section, `add_rec_session` is:
`add_rec_session n/STUDENT_NAME d/START_DATE e/END_DATE b/INTERVAL t/TIME k/DURATION s/SUBJECT f/FEE`
In the summary, it's the one you provided. probably gotta decide on one?

Reply:
oh yes thanks for catching this. I think we should have it as START_DATE instead of just DATE